src/lib/__tests__/sortMenu-test.js

```diff
@@ -0,0 +1,176 @@
+import sortMenu from '../sortMenu';
+
+describe('sortMenu', () => {
+  it('modifies original array to follow the group, then name order provided', () => {
+    const initialArray = [
+      {props: {group: 'DEV', name: 'Inspector'}},
+      {props: {group: 'DEV', name: 'JSON'}},
+    ];
+    const orderProp = [{group: 'DEV', name: 'JSON'}, {group: 'DEV', name: 'Inspector'}];
+    sortMenu(initialArray, orderProp);
+
+    expect(initialArray[0].props.name).toBe('JSON');
```

Comment:
I would express these as `expect(initialArray).toBe( ... the full array ... )` ... More readable and more complete IMO :)